test/cmdlineTests.sh

```diff
@@ -275,12 +275,18 @@ EOF
     sed -i.bak -e 's/^\(Dynamic exception type:\).*/\1/' "$stderr_path"
     rm "$stderr_path.bak"
 
-    if [[ $exitCode -ne "$exit_code_expected" ]]
+    if [[ "$(cat "$stderr_path")" != "${stderr_expected}" ]]
```

Comment:
Note that while the diff may at first look like there are more changes here, I'm really only changing the order of the `if`s.